src/cdk/scrolling/virtual-scroll-viewport.spec.ts

```diff
@@ -973,15 +1002,15 @@ class FixedSizeVirtualScroll {
   // Casting virtualForOf as any so we can spy on private methods
   @ViewChild(CdkVirtualForOf, {static: true}) virtualForOf: any;
 
-  @Input() orientation = 'vertical';
-  @Input() viewportSize = 200;
-  @Input() viewportCrossSize = 100;
-  @Input() itemSize = 50;
-  @Input() minBufferPx = 0;
-  @Input() maxBufferPx = 0;
-  @Input() items = Array(10).fill(0).map((_, i) => i);
-  @Input() trackBy: TrackByFunction<number>;
-  @Input() templateCacheSize = 20;
+  orientation = 'vertical';
```

Comment:
Removing the `Input` here isn't necessary for the fix, but I noticed that they don't actually do anything.